source/code-type/pocos.txt

```diff
@@ -0,0 +1,104 @@
+.. _mongodb-analyzer-analyze-pocos:
+
+=============
+Analyze POCOs
+=============
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+":wikipedia:`Plain old CLR/class objects <Plain_old_CLR_object>`", or **POCOs**, are
+simple class objects that do not
```

Comment:
I noticed this rule isn't in the style guide, but came from another style guide so I will remove it. https://jira.mongodb.org/browse/DOCSP-33112